processor/setup_law_remote.sh

```diff
@@ -8,6 +8,13 @@ action(){
         [ ! -z "$1" ] && export PATH="$1:${PATH}" && echo "Add $1 to PATH"
     }
 
+    echo "------------------------------------------"
+    echo " | HOSTNAME = $(hostname)"
+    echo " | ANA_NAME = ${ANA_NAME}"
+    echo " | ENV_NAME = ${ENV_NAME}"
+    echo " | TARBALL_PATH = ${TARBALL_PATH}"
+    echo "------------------------------------------"
```

Comment:
This seems a bit out of order. The "line" should be after the paths in line 33-40

Reply:
agreed